docs/content/annotations.md

```diff
@@ -203,15 +203,149 @@ allow {
 
 ### Scope
 
-The `scope` annotation determines how the `schema` annotation will be applied.
-If the `scope` annotation is omitted, it defaults to the scope for the statement that immediately follows the annotation.
-In-depth information on this topic can be found [here](../schemas#annotation-scopes).
+Annotations can be defined at the rule or package level. The `scope` annotation in
+a metadata block determines how that metadata block will be applied. If the
+`scope` field is omitted, it defaults to the scope for the statement that
+immediately follows the annotation. The `scope` values that are currently
+supported are:
+
+* `rule` - applies to the individual rule statement. Default, when metadata block precedes rule.
+* `document` - applies to all of the rules with the same name in the same package
+* `package` - applies to all of the rules in the package, Default, when metadata block precedes package.
+* `subpackages` - applies to all of the rules in the package and all subpackages (recursively)
+
+In case of overlap, schema annotations override each other as follows:
+
+```
+rule overrides document
+document overrides package
+package overrides subpackages
+```
```

Comment:
💭 As long as we don't end up with 15 levels of precedence...  Chef had that 👇 and it's been the source of every second support ask.

![chef](https://docs.chef.io/images/overview_chef_attributes_table.png)

Reply:
Let's hope not 😬. But this is nothing new; this behavior is adopted from how `scope` applied to the `schemas` annot. A different behavior for other annotations would get very confusing.